### PR TITLE
added to Sols html and did some CSS

### PR DESCRIPTION
**Pull request by BenNewman90**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
+
+
+/* mobile first */
```
Comment (tawfiknasser) [inline style.css]: :+1: 
Mobile->Tablet->Desktop

File: style.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+.barChart {
+  width: 80%;
+}
+.map {
+  width: 80%;
+}
+
+/* Desktop */
```
Comment (tawfiknasser) [inline style.css]: desktop i think it's  min 1000 px
mobile<600
tablet 600-1000
desktop 1000+

What do you think ?

**Comment by tawfiknasser** [comment]
might you also please reference the issue number = #14  when committing ?
:1st_place_medal:

File: style.css (hunk 2 above)
Comment (DenisKent) [reply style.css]: The cut-offs I have been using are mobile:<480px, tablet <800, desktop 800+. But I supose the exact limits don't matter much as long as it looks good accross the board

**Pull request merged**